### ISSUE TITLE
When entering Chinese in Neovide, the input method indicates that the floating layer cannot track the cursor

### ISSUE THREAD

**Issue by asins** [neovide/neovide#1721]
**Describe the bug**

Hello, this is using the Chinese input method to enter text, but the input method prompts the layer one true solution in the lower left. After entering the full screen, the IME hint layer is not visible.

**To Reproduce**
Just open the NeoVide app and use a Chinese input method.

**Expected behavior**
1. When entering text in non-full-screen mode, the IME hint layer can follow the position of the cursor. 
3. In full-screen mode, you can see the input method hint layer and follow the cursor position.

**Screenshots**
This is problematic under the input float not following the cursor position：

https://user-images.githubusercontent.com/898354/210521449-6b58037c-bc0b-4d2b-be1d-167766c32d02.mov

When used by other editors, the input method is displayed normally.

https://user-images.githubusercontent.com/898354/210523242-0d64258d-db42-4732-a88f-1eb0dde4e3c9.mov

**Desktop (please complete the following information):**
 - OS: macOS 13.1
 - Neovide Version 0.10.0
 - Neovim Version 0.8.2

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**

**Additional context**

**Comment by 3bluebird** [comment]
Have the same problem, worrying 😣

**Comment by swithun-liu** [comment]
same problem

1. the IME hint layer should follow the cursor
2. when I click back, it both delete the text before the cursor and the text in the IME hint layer

**Comment by c-my** [comment]
Have the same annoying problem on Windows 😞

**Comment by namespacebilibili** [comment]
waiting..

**Comment by jensenojs** [comment]
waiting(

**Comment by fredizzimo** [comment]
This should be implemented in https://github.com/neovide/neovide/pull/1899 

But there's still a bug when you are close to the bottom of the screen, this will be fixed once this is merged and we take it into use https://github.com/rust-windowing/winit/pull/2889

**Comment by fredizzimo** [comment]
This bug is actually a duplicate of #1127, so I'm closing this one.

The overlapping issue has now been fixed in #1899, so it should be completely fixed there. But it would be nice if someone could try it out there. 

NOTE, IME support on X11, might be completely broken on X11 at the moment, see #1006. But on other platforms and on Wayland it should work. But if you do get it to work on X11, please report it there, so that we can track possible configuration issues.

**Comment by xpbluesmile** [comment]
I have installed the font, but I am using the X11 platform and it still cannot be displayed correctly.
![2024-03-19_16-55](https://github.com/neovide/neovide/assets/3887637/d98c7e0e-1ff3-4e48-bfdf-429182216488)

**Comment by xpbluesmile** [comment]
![Uploading 2.png…]()

**Comment by asins** [comment]
Indeed, this issue has been fixed.
